recipes/cuda-opencl/build.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+# Install to conda style directories
+[[ -d lib64 ]] && mv lib64 lib
+mkdir -p ${PREFIX}/lib
+[[ -d pkg-config ]] && mv pkg-config ${PREFIX}/lib/pkgconfig
+[[ -d "$PREFIX/lib/pkgconfig" ]] && sed -E -i "s|cudaroot=.+|cudaroot=$PREFIX|g" $PREFIX/lib/pkgconfig/opencl*.pc
+
+[[ ${target_platform} == "linux-64" ]] && targetsDir="targets/x86_64-linux"
+[[ ${target_platform} == "linux-ppc64le" ]] && targetsDir="targets/ppc64le-linux"
+[[ ${target_platform} == "linux-aarch64" ]] && targetsDir="targets/sbsa-linux"
+
+for i in `ls`; do
+    [[ $i == "build_env_setup.sh" ]] && continue
+    [[ $i == "conda_build.sh" ]] && continue
+    [[ $i == "metadata_conda_debug.yaml" ]] && continue
+    if [[ $i == "lib" ]] || [[ $i == "include" ]]; then
+        # Headers and libraries are installed to targetsDir
+        mkdir -p ${PREFIX}/${targetsDir}
+        mkdir -p ${PREFIX}/$i
+        cp -rv $i ${PREFIX}/${targetsDir}
+        # Don't symlink libs in $PREFIX/lib to avoid clobbering
+        #if [[ $i == "lib" ]]; then
+        #    for j in "$i"/*.so*; do
+        #        # Shared libraries are symlinked in $PREFIX/lib
+        #        ln -s ${PREFIX}/${targetsDir}/$j ${PREFIX}/$j
+        #    done
+        #fi
```

Comment:
Should we delete this instead of commenting it out?

Reply:
Sure, I left the descriptive comment as a reminder that this is intentional.